File: C++/006. Universidade/Universidade.cpp
#include <iostream>
int main()
{
    int tempo;
    std::string universidade;

    std::cin >> universidade >> tempo;
    std::cout << "Voce estuda na " << universidade << " ha " << tempo << " anos." << std::endl;
    return 0;
}
// Compile with: g++ -o Universidade Universidade.cpp
// Run with: ./Universidade

File: Java/006. Universidade/Universidade.java
import java.util.Scanner;

public class Universidade
{
    public static void main(String[] args)
    {
        int tempo;
        String universidade;
        Scanner scan = new Scanner(System.in);
        universidade = scan.next();
        tempo = scan.nextInt();
        System.out.println("Voce estuda na " + universidade + " ha " + tempo + " anos.");
        scan.close();       
    }
}
